### DISCUSSION TITLE
Ability to include nested documents in download

### DISCUSSION THREAD

**Discussion by ChuckJonas** [outline/outline#6822] [Ideas]
It would be extremely helpful if when downloading (or printing) a document, there was an option to include nested documents.

**Example Usecase:** 
We are maintaining our security policy in outline.  For better organization, we are using nested documents.  However, we often have to provide this to a new client for review or for employees to sign, and there isn't currently any easy way to get all the files out into a single document.

**Comment by tommoor** [comment]
Yes, we should add this – concatenating into a single PDF adds a layer of difficulty

**Reply by ChuckJonas** [reply]
Even if the content was just "Flat Mapped` in recursive descent order, it would be good enough...

EG This nested structure:

 ```
- Document 1
-- Document 1.1
--- Document 1.1.1
--- Document 1.1.2
-- Document 1.2
--- Document 1.2.1
```
Would append the context of all the documents in the following order:

```
- Document 1
- Document 1.1
- Document 1.1.1
- Document 1.1.2
- Document 1.2
- Document 1.2.1
```

It would be nice if there was a top level TOC and potentially some type of header structure, but it's not absolutely needed.
